firebase-firestore/firebase-firestore.gradle

```diff
@@ -64,15 +65,32 @@ android {
         multiDexEnabled true
         testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
         consumerProguardFiles 'proguard.txt'
+
+        if (findProperty("useProdBackendForTests")) {
+            buildConfigField("boolean", "USE_EMULATOR_FOR_TESTS", "false")
+        } else {
+            buildConfigField("boolean", "USE_EMULATOR_FOR_TESTS", "true")
+        }
+
+        if (hasProperty('targetBackend')) {
+            buildConfigField("String", "TARGET_BACKEND", "\"${property("targetBackend")}\"")
+        } else {
+            // By default set the target backend to 'emulator'
+            buildConfigField("String", "TARGET_BACKEND", "\"emulator\"")
+        }
+        def localProps = new Properties()
```

Comment:
This isn't being used; we should just drop it. Probably a remnant from the past.